### PR TITLE
APP-187 Implemented python tool to build C/C++ sample applications

### PR DESCRIPTION
**Pull request by vadimol**

### CODE REVIEW

File: scripts/apptester/apptester.py [hunk 1]
```diff
@@ -0,0 +1,294 @@
+#!/usr/bin/env python
```
Comment (RostakaGmfun) [inline scripts/apptester/apptester.py:1]: Add licence header

File: scripts/apptester/apps.yam [hunk 2]
```diff
@@ -0,0 +1,116 @@
+# This file contains configurations to build sample application
```
Comment (RostakaGmfun) [inline scripts/apptester/apps.yam]: Add licence header

File: scripts/apptester/apptester.py [hunk 3]
```diff
@@ -0,0 +1,294 @@
...
+            for language in languages:
+                platforms = languages[language]['platform']
+                if language == 'c':
+                    lang = KaaSDKLanguage.C
+                elif language == 'cpp':
+                    lang = KaaSDKLanguage.CPP
+                elif language == 'java':
+                    lang = KaaSDKLanguage.JAVA
+                else:
+                    lang = KaaSDKLanguage.OBJC
```
Comment (RostakaGmfun) [inline scripts/apptester/apptester.py]: Add `elif language == 'objc':` and some error handling for anything else.

File: scripts/apptester/apptester.py (hunk 1 above)
Comment (vadimol) [inline scripts/apptester/apptester.py:1]: done, thanks

File: scripts/apptester/apps.yam (hunk 2 above)
Comment (vadimol) [inline scripts/apptester/apps.yam]: done, thanks

File: scripts/apptester/apptester.py (hunk 3 above)
Comment (vadimol) [inline scripts/apptester/apptester.py]: fixed, thanks

File: scripts/apptester/apptester.py [hunk 4]
```diff
@@ -0,0 +1,312 @@
...
+        # currently SDK path is hardcoded in application's cmake file
+        sdkdir = os.path.join(self.config.buildpath, 'libs/kaa')
+        sdkfile = os.path.join(sdkdir, 'kaa-%s-sdk.tar.gz'% \
+                               (self.config.language.lower()))
+        os.makedirs(sdkdir)
+
+        profiles = self.kaanode.get_sdk_profiles(self.config.name,
+                                                 self.kaauser)
+
+        # For now all sample applications use only one SKD profile.
```
Comment (RostakaGmfun) [inline scripts/apptester/apptester.py]: SDK

File: scripts/apptester/apptester.py [hunk 5]
```diff
@@ -0,0 +1,312 @@
...
+            for language in languages:
+                platforms = languages[language]['platform']
+                if language == 'c':
+                    lang = KaaSDKLanguage.C
+                elif language == 'cpp':
+                    lang = KaaSDKLanguage.CPP
+                elif language == 'java':
+                    lang = KaaSDKLanguage.JAVA
+                elif language == 'objc':
+                    lang = KaaSDKLanguage.OBJC
```
Comment (RostakaGmfun) [inline scripts/apptester/apptester.py]: `OBJECTIVE_C`

File: scripts/apptester/apptester.py (hunk 4 above)
Comment (vadimol) [inline scripts/apptester/apptester.py]: Fixed

File: scripts/apptester/apptester.py (hunk 5 above)
Comment (vadimol) [inline scripts/apptester/apptester.py]: Fixed, thanks

**Comment by RostakaGmfun** [comment]
LGTM

**Pull request merged**